src/components/ChatLog.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = (props) => { 
+       
+    const chatDataMap = props.chats.map(item =>(
+    <ChatEntry 
+        key={item.id}
+        id={item.id} 
+        sender={item.sender} 
+        body={item.body} 
+        timeStamp={item.timeStamp} 
+        liked={item.liked} 
+        onUpdateChat={props.onUpdateChat}/> ))
+
+        return (       
+            <main>
+            {chatDataMap}
+            {/* Wave 01: Render one ChatEntry component
+            Wave 02: Render ChatLog component */}
+            </main>
+                
+ ); }
+
+ChatLog.propTypes = {
+    chats:PropTypes.arrayOf(PropTypes.shape({
```

Comment:
great use of shape. Make sure to only had isRequired to props that are a must for the component to render. If you want them all to be required then you should update the tests to reflect that.